src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -200,6 +204,7 @@ public GameService(ClientProperties clientProperties,
         item -> new Observable[]{item.statusProperty(), item.getTeams()}
     );
     JavaFxUtil.attachListToMap(games, uidToGameInfoBean);
+    JavaFxUtil.addListener(gameRunning, (observable, oldValue, newValue) -> reconnectService.setGameRunning(newValue));
```

Comment:
I'm a bit confused. We have two services talking to each other. No UI involeved. Why are we using JavaFxUtil here? We should keep out JavaFx stuff wherever possible, don't we?

Reply:
It is for thread safety. Won't harm and should always be used